src/generators/text-style.ts

```diff
@@ -1,10 +1,27 @@
 import type {FilterConfig} from '../definitions';
 
 export function createTextStyle(config: FilterConfig): string {
-    const lines: string[] = [];
     // Don't target pre elements as they are preformatted element's e.g. code blocks
     // Exclude font libraries to preserve icons
-    lines.push('*:not(pre, pre *, code, .far, .fa, .glyphicon, [class*="vjs-"], .fab, .fa-github, .fas, .material-icons, .icofont, .typcn, mu, [class*="mu-"], .glyphicon, .icon) {');
+
+    const mono: string = [
```

Comment:
```suggestion
    const exlcudingSelectors: string = [
```
Should be a bit more verbose. `mono` is vague to me.